http/http-advanced-reactive/src/main/java/io/quarkus/ts/http/advanced/reactive/Person.java

```diff
@@ -0,0 +1,261 @@
+package io.quarkus.ts.http.advanced.reactive;
+
+import java.util.List;
+
+import io.quarkus.runtime.annotations.RegisterForReflection;
+
+@RegisterForReflection
+public class Person {
+
+    private String _id;
+    private int index;
+    private String guid;
+    private boolean isActive;
+    private String balance;
+    private String picture;
+    private int age;
+    private String eyeColor;
+    private String name;
+    private String gender;
+    private String company;
+    private String email;
+    private String phone;
+    private String address;
+    private String about;
+    private String registered;
+    private double latitude;
+    private double longitude;
+    private List<String> tags;
+    private List<Friend> friends;
+    private String greeting;
+    private String favoriteFruit;
+
+    public Person() {
+    }
+
+    public Person(String _id, int index, String guid, boolean isActive, String balance, String picture, int age,
+            String eyeColor, String name, String gender, String company, String email, String phone, String address,
+            String about, String registered, double latitude, double longitude, List<String> tags, List<Friend> friends,
+            String greeting, String favoriteFruit) {
+        this._id = _id;
+        this.index = index;
+        this.guid = guid;
+        this.isActive = isActive;
+        this.balance = balance;
+        this.picture = picture;
+        this.age = age;
+        this.eyeColor = eyeColor;
+        this.name = name;
+        this.gender = gender;
+        this.company = company;
+        this.email = email;
+        this.phone = phone;
+        this.address = address;
+        this.about = about;
+        this.registered = registered;
+        this.latitude = latitude;
+        this.longitude = longitude;
+        this.tags = tags;
+        this.friends = friends;
+        this.greeting = greeting;
+        this.favoriteFruit = favoriteFruit;
+    }
+
+    public String get_id() {
```

Comment:
if you verify a JSON payload, you are probably interested in commonly used features like ignored field etc. Anyway, that's a side note. Why is this called `get_id`? I wonder into which field is it going to be turned by a serializer. Is it really going to be `id` field? You might want to check.